packages/@webex/plugin-cc/src/services/agent/types.ts

```diff
@@ -1,7 +1,7 @@
 import {Msg} from '../core/GlobalTypes';
 
 export type LogoutSuccess = Msg<{
-  eventType: 'AgentDesktopMessage';
```

Comment:
Why have we made the eventType less specific for all the types?

Reply:
as discussed, added back